src/resource-group-controller_impl.h

```diff
@@ -15,41 +15,55 @@
 #include "resource-group-controller.h"
 #include "resource-group-view.h"
 #include "resource-group.h"
+#include "resource-manager.h"
 
 #include "log.h"
 
 NAMESPACE__THITTAM__START
 
-class ResourceGroupControllerImpl :
-  public ResourceGroupController,
+class ResourceGroupControllerImpl : public ResourceGroupController,
```

Comment:
If you cannot have all superclasses in the same line, then indent all of them.

Reply:
That is indented.